hw/ip/prim/dv/prim_lfsr/data/prim_lfsr_cover.cfg

```diff
@@ -0,0 +1,9 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifer: Apache-2.0
+
++tree prim_lfsr_tb.gen_duts[24].i_prim_lfsr
+-module clk_rst_if
```

Comment:
Is this instantiated in `i_prim_lfsr` instance? Otherwise it should not be needed.

Reply:
whoops, you're right! this was an artifact of my original coverage approach and should be removed.